TSKit.xcodeproj/project.pbxproj

```diff
@@ -668,7 +670,7 @@
 				HEADER_SEARCH_PATHS = "$(PROJECT_DIR)/TSKit/include";
 				INFOPLIST_FILE = TSKit/Info.plist;
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.2;
```

Comment:
Is there a particular reason you change the deployment target to iOS 12?

Reply:
No specific reason just habit. I can switch it back but I would also ask any specific reason to leave it a 10.3? Just out of curiosity.

Reply:
The projects using the library would be forced to target at least the same iOS version. Even if the iOS 12 adoption is quite high, I'd keep the library deployment low